### PR TITLE
Version 0.1.1. Allow no_std.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,13 +1,18 @@
 //! # Wrappers for compatibility with rand 0.4
 
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
```
Comment (mbr) [inline src/lib.rs]: `PhantomData` is also available in `core`, we could just use that here and lose the compiler directive, I *think*.

Would that make it feasible to offer these wrappers for std and non-std alike?

Comment (mbr) [reply src/lib.rs]: There's different conventions on how to handle the std vs non-standard variants, either importing everything from core or doing something like https://docs.rs/rand/0.4.3/src/rand/lib.rs.html#250. I have no idea which one is *en vogue* at the moment.

**Review by mbr — COMMENTED** [review]
At first glance, it looks like it should be possible to compile this crate completely with `no_std`, what's keeping it back at the moment?

File: src/lib.rs (hunk 1 above)
Comment (afck) [reply src/lib.rs]: You're right, thanks! I amended the commit.

**Comment by mbr** [comment]
Alright, LGTM (I'm missing write access, so I can't sign off on these changes or merge them).

**Pull request merged**